Shared/Samples/Create buffers around points/CreateBuffersAroundPointsView.swift

```diff
@@ -0,0 +1,303 @@
+            let (points, radii) = bufferPoints.reduce(into: ([Point](), [Double]())) { (result, pointAndRadius) in
+                result.0.append(pointAndRadius.point)
+                result.1.append(pointAndRadius.radius)
+            }
+            
+            // Create the buffers.
+            // Notice: the radius distances has the same unit of the map's spatial reference's unit.
+            // In this case, the statePlaneNorthCentralTexas spatial reference uses US feet.
+            let bufferPolygon = GeometryEngine.buffer(around: points, distances: radii, shouldUnion: shouldUnion)
+            
+            // Add the tap points to the tapPointsGraphicsOverlay.
+            points.forEach { point in
+                tapPointsGraphicsOverlay.addGraphic(Graphic(geometry: point))
+            }
+            
+            // Add the buffers to the bufferGraphicsOverlay.
+            bufferPolygon.forEach { buffer in
+                bufferGraphicsOverlay.addGraphic(Graphic(geometry: buffer))
+            }
+        }
+        
+        /// Add a point with its radius to the bufferPoints array.
+        func addBufferPoint() {
+            // Check to ensure the tapPoint is within the boundary
+            guard isWithinBounds(tapPoint) else {
+                status = .outOfBoundsTap
+                return
+            }
+            
+            // Ensure that there is input.
+            if let radius = Double(radiusInput) {
+                // Ensure that the input is valid.
+                guard radius > 0 && radius < 300 else {
+                    status = .invalidInput
+                    return
+                }
+                
+                // Update the buffer radius with the text value.
+                let radiusInMiles = Measurement(value: radius, unit: UnitLength.miles)
+                
+                // The spatial reference in this sample uses US feet as its unit.
+                let radiusInFeet = radiusInMiles.converted(to: .feet).value
+                
+                // Add point with radius to bufferPoints Array.
+                bufferPoints.append((point: tapPoint!, radius: radiusInFeet))
+                status = .bufferCreated
+            }
+        }
+        
+        /// Clear the bufferPoints array and related graphics.
+        func clearBufferPoints() {
+            bufferPoints.removeAll()
+            bufferGraphicsOverlay.removeAllGraphics()
+            tapPointsGraphicsOverlay.removeAllGraphics()
+        }
+    }
+}
+
+private extension CreateBuffersAroundPointsView {
+    /// An enumeration for the sample status.
+    enum Status {
+        case addPoints
+        case bufferCreated
+        case outOfBoundsTap
+        case invalidInput
+        case noPoints
+    }
+    
+    /// Gets the text message associated with the current status for the overlay.
+    /// - Returns: A 'String' status message.
+    func getStatusText(_ status: Status) -> String {
```

Comment:
We typically put this descriptive string as part of the enum itself, i.e.,

See an example in `ShowResultOfSpatialOperationsView.SpatialOperation`

```swift
        var label: String {
            switch self {
            case .addPoints:
                return "Tap on the map to add buffers."
            case .bufferCreated:
                return "Buffer created."
            case .outOfBoundsTap:
                return "Tap within the boundary to add buffer."
            case .invalidInput:
                return "Enter a value between 0 and 300 to create a buffer."
            case .noPoints:
                return "Add a point to draw the buffers."
            }
        }
```